### DISCUSSION TITLE
error (Module '"@remix-pwa/dev"' has no exported member 'WorkerDataFunctionArgs')

### DISCUSSION THREAD

**Discussion by ToyQiv** [ShafSpecs/remix-pwa#153] [Q&A]
error in my "entry.workers.ts" (Module '"@remix-pwa/dev"' has no exported member 'WorkerDataFunctionArgs')
I am trying the 
https://remix-pwa.run/docs/main/default-fetch-handler
--
import type { WorkerDataFunctionArgs } from '@remix-pwa/dev';

export const defaultFetchHandler = async ({
  request
}: WorkerDataFunctionArgs) => {
--

any suggestions

**Comment by MukundaKatta** [comment]
The `WorkerDataFunctionArgs` type was renamed to `DefaultFetchHandlerArgs` in `@remix-pwa/dev` around v3.1 — the docs page you're on still uses the old name. Swap the import:

```ts
import type { DefaultFetchHandlerArgs } from '@remix-pwa/dev';
export const defaultFetchHandler = async ({ request }: DefaultFetchHandlerArgs) => { … }
```

If you're on an older `@remix-pwa/dev`, `WorkerDataFunctionArgs` still works — a mismatch between installed version and the docs page version is the usual cause.
